org.civicrm.reqangular/src/common/models/option-group.js

```diff
@@ -1,28 +1,31 @@
+/* eslint-env amd */
+
 define([
-    'common/modules/models',
-    'common/models/model',
-    'common/services/api/option-group'
+  'common/modules/models',
+  'common/models/model',
+  'common/services/api/option-group'
 ], function (models) {
-    'use strict';
-
-    models.factory('OptionGroup', [
-        'Model', 'api.optionGroup',
-        function (Model, optionGroupAPI) {
+  'use strict';
 
-            return Model.extend({
+  models.factory('OptionGroup', [
+    'Model', 'api.optionGroup',
+    function (Model, optionGroupAPI) {
+      return Model.extend({
 
-                /**
-                 * Returns the values of a sigle or multiple option groups
-                 *
-                 * @param {string/array} names
-                 *   Bases on the type of the parameter, the method will return
-                 *   either an array of values (string) or an object (array)
-                 * @return {Promise}
-                 */
-                valuesOf: function (names) {
-                    return optionGroupAPI.valuesOf(names);
-                }
-            });
+        /**
+         * Returns the values of a sigle or multiple option groups
+         *
+         * @param  {String|Array} names
+         *   Bases on the type of the parameter, the method will return
+         *   either an array of values (string) or an object (array)
+         * @param  {Object} params optional parameters for the query
+         * @param  {Boolean} cache optional parameter to cache the query or not
```

Comment:
you can mark optional parameters with `[]` (http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values)

Reply:
Done.